### ISSUE TITLE
As an LMA Admin I want a trigger to populate the Lead.License field when a License is created so that the XOF fields from another story populate

### ISSUE THREAD

**Issue by jengler74** [CodeScience/BOSS#18]

**Comment by ronkiker** [comment]
- license created
- go to parent lead -> put in License ID
- once this is complete --> this will populate the XOF fields that go from License to Lead. --> Lead creation from LMA, we need to send the values from Lead to License, thereby creating a two way relations (license becomes parent to lead, in addition to being a child to it)
